cart diff with older version

diff --git a/openstreetmap-carto.style b/openstreetmap-carto.style
index 436bc59..ea366b7 100644
--- a/openstreetmap-carto.style
+++ b/openstreetmap-carto.style
@@ -64,6 +64,8 @@ node,way   sport        text         polygon
 node,way   surface      text         linear
 way   source:geometry:date     text    linear
 way   source:geometry:oidn     text    linear
+way   source:geometry:uidn     text    linear
+way   source:geometry:entity   text    linear
 way   source:geometry  text    linear
 node,way   toll         text         linear
 node,way   tourism      text         polygon
@@ -96,6 +98,12 @@ way         area:highway            text    phstore
 # These tags are used by mappers to keep track of data.
 
 # They aren't very useful for rendering. (but usefull for what we want to do with it)
+node,way    LENGTE                  text    delete
+node,way    OPPERVL                 text    delete
+#node,way    TYPE                   text    delete
+#node,way    UIDN                   text    delete
+node,way    LBLVORM                 text    delete
+node,way    VORM                    text    delete
 node,way    note                    text    delete
 node,way    note:*                  text    delete
 node,way    source                  text    linear

